OnlineLibrary.API/Controllers/AuthorController.cs

```diff
@@ -12,7 +15,9 @@ namespace OnlineLibrary.API.Controllers
     public class AuthorController : ControllerBase
     {
         private readonly IAuthorService _authorService;
-
+        
+        private static readonly ILog _logger = LogManager.GetLogger(typeof(AuthorController));
```

Comment:
remove and pass ILogger<> to constructore, apply this logic in all places where you need logger